python_modules/libraries/dagster-dg/dagster_dg_tests/cli_tests/test_vscode_command.py

```diff
@@ -30,12 +31,32 @@ def test_configure_editor(editor: str) -> None:
 
         assert out.exit_code == 0
 
-        sample_project_vscode_path = Path.cwd() / ".vscode"
-        assert (sample_project_vscode_path / "schema.json").exists()
+        sample_project_dg_path = Path.cwd() / ".dg"
```

Comment:
Changed this to `.dg` folder instead of `.vscode`, open to other suggestions though. We'll likely want a "place to store dg metadata" in code locations at some point for other commands, I'd reckon?